requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 # Testing
-pytest==4.0.2
-pytest-cov==1.8.1
-coverage==3.7.1
+pytest==4.6.6
+pytest-cov==2.6.1
+coverage==4.5.4
```

Comment:
Upgrading the test dependencies is needed because a transitive dependency broke (same error as https://stackoverflow.com/questions/58189683/typeerror-attrib-got-an-unexpected-keyword-argument-convert)